app/policies/category_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CategoryPolicy < ApplicationPolicy
+  attr_reader :user, :record
```

Comment:
We can remove that. because inherited from the `ApplicationPolicy` and able to use `@user @record`


```
 def initialize(user, record)
    @user = user
    @record = record
  end
```

Reply:
agree